### PR TITLE
Add -y option to the update command

### PR DESCRIPTION
**Pull request by TomasHofman**
Fixes https://github.com/wildfly-extras/prospero/issues/28.

### CODE REVIEW

**Comment by spyrkob** [comment]
This breaks `UpdateCommandTest` - I think just mock expectations need to be changed

**Comment by TomasHofman** [comment]
Should be fixed now.

File: prospero-common/src/main/resources/org/wildfly/prospero/Messages.i18n_cs_CZ.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+dirMustBeDirectory=Daná cesta %s je soubor. Je třeba prázdný adresář.
```
Comment (spyrkob) [inline prospero-common/src/main/resources/org/wildfly/prospero/Messages.i18n_cs_CZ.properties:1]: Wrong PR?

Comment (TomasHofman) [reply prospero-common/src/main/resources/org/wildfly/prospero/Messages.i18n_cs_CZ.properties]: Yes, that's not supposed to be there. Updated.

**Review by spyrkob — APPROVED** [review]

**Pull request merged**